holoviews/plotting/bokeh/plot.py

```diff
@@ -153,7 +154,43 @@ def sync_sources(self):
 
 
 
-class GridPlot(BokehPlot, GenericCompositePlot):
+class CompositePlot(BokehPlot):
+
```

Comment:
Would be good to have a docstring explaining what this base class is for. Right now, everything ``CompositePlot`` does seems to be only needed by ``LayoutPlot``. Any ideas what ``CompositePlot`` could be used for that isn't ``LayoutPlot``?